README.md

```diff
@@ -1,2 +1,88 @@
-# Weather App
+<h1 align="center">[Javascript] WeatherApp </h1>
+<p>
+  <img alt="Version" src="https://img.shields.io/badge/version-0.0.1-blue.svg?cacheSeconds=2592000" />
+  <a href="#" target="_blank">
+    <img alt="License: MIT " src="https://img.shields.io/badge/License-MIT -yellow.svg" />
+  </a>
+  <a href="https://twitter.com/ae_cordova" target="_blank">
+    <img alt="Twitter: ae_cordova " src="https://img.shields.io/twitter/follow/ae_cordova .svg?style=social" />
+  </a>
+</p>
 
+
+___
+## Description of the project
+
+The purpose of this project is to practice Javascript acquired knowledge by building a web app that displays the current weather of a city, either on C or F degree scale.
+
+![Screenshot](./screenshot.png?raw=true "Screenshot")
+
+### ⭐️ Built with
+* ES6
+* HTML5/CSS3
+* Webpack
+* npm
+* Openweather API
+
+### Dev Environment Requirements:
+* To run this code locally you must have npm, then run  ```npm install```  to add all the required packages and dependencies.
+### ⭐️ Running the code
+
+#### Live Version:
+
+You can find a live running version of this app on the following link: [Restaurant Page](https://js-weatherapp.netlify.app/)
```

Comment:
kindly correct the name of the live link
weather app not restaurant page right?

Reply:
Corrected